pkg/kubelet/allocation/allocation_manager_test.go

```diff
@@ -173,8 +197,23 @@ func TestUpdatePodFromAllocation(t *testing.T) {
 		},
 		expectUpdate: true,
 		expectPod:    resizedPod,
+	}, {
+		name: "with resized pod-level resource allocation",
```

Comment:
Can we also have a cases with the feature gate off, and a case with just pod level resources (no container resources specified)?